Header header

float64 count
float64 pulses_sec
float64 revolutions_sec
float64 velocity